### PR TITLE
Pass meta data into native methods

### PR DESCRIPTION
**Pull request by jkrems**
This allows picking up this data when instrumenting an application.

### CODE REVIEW

**Comment by jkrems** [comment]
Added a commit to fix a problem with `util` not being found in React Native.

File: test/instrument.browser.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+var assign = require('lodash/assign');
+
+var original = typeof window !== 'undefined' ? window.fetch : null;
```
Comment (dbushong) [inline test/instrument.browser.js]: Is the existence of `window` even worth checking?

**Comment by dbushong** [comment]
nit, otherwise :sheep: it

File: test/instrument.browser.js (hunk 1 above)
Comment (jkrems) [inline test/instrument.browser.js]: Yeah, `mocha` will blindly load all files in `test/*` even when running in node... :/

**Pull request merged**